playbooks/openshift-checks/health.yml

```diff
@@ -0,0 +1,15 @@
+---
+- include: ../openshift-cluster/initialize_groups.yml
+
+- name: Run OpenShift health checks
+  hosts: OSEv3
+  # Temporarily reverting to OSEv3 until group standardization is complete
```

Comment:
I think we can get rid of this line, it explains something out of context, that only makes sense if we go look at the Git history.